tests/suites/test_suite_pk.function

```diff
@@ -200,7 +200,9 @@ void invalid_parameters( )
     unsigned char *null_buf = NULL;
```

Comment:
This is only needed if `PK_PARSE_C` is defined and will lead to an unused variable warning otherwise. I'd suggest to add `((void) xxx)` statements throughout to guard against such issues.

Reply:
The same holds for `p` as far as I see.